label_maker/label.py

```diff
@@ -66,99 +69,124 @@ def make_labels(dest_folder, zoom, country, classes, ml_type, bounding_box, spar
         Other properties from CLI config passed as keywords to other utility functions
     """
 
-    mbtiles_file = op.join(dest_folder, '{}.mbtiles'.format(country))
-    mbtiles_file_zoomed = op.join(dest_folder, '{}-z{!s}.mbtiles'.format(country, zoom))
-
-    if not op.exists(mbtiles_file_zoomed):
-        filtered_geo = kwargs.get('geojson') or op.join(dest_folder, '{}.geojson'.format(country))
```

Comment:
Here, I am not sure I understand why kwargs.get('geojson') is picked up here for the variable. This would assume that the geojson provided in the `config.json` file contains the label features on which to generate the labelled output for classification, object detection or segmentation.

However, I cannot see a use case where someone would provide their labelled features as a standalone geojson file, because then would they really need to use the label-maker?
If we keep the `geojson` attribute in its current defintion, I still see the value of an `aoi` field to provide the AOI in the form of geometries, as an alternative to the `country+bbox `combination.

Reply:
@wouellette the `geojson` attribute is provided so users can add a standalone GeoJSON file. `label-maker` still provides value here in that it will chip up the imagery and GeoJSON labels, rasterize or create object labels, etc. We need to keep this attribute as it's used in a number of existing workflows.